apisix/core/config_etcd.lua

```diff
@@ -961,7 +979,9 @@ function _M.new(key, opts)
     if not health_check_timeout or health_check_timeout < 0 then
         health_check_timeout = 10
     end
-
+    if key then
+        sync_status_to_shdict(true)
+    end
```

Comment:
What does the fact that the key is not null mean? You may need to explain this by adding some additional comments, I don't understand what this is.